src/main/java/com/ldtteam/blockout/controls/ItemIcon.java

```diff
@@ -106,13 +113,84 @@ public void drawSelf(final MatrixStack ms, final double mx, final double my)
             renderItemModelIntoGUI(itemStack,
                 ms,
                 Minecraft.getInstance().getItemRenderer().getModel(itemStack, null, Minecraft.getInstance().player));
```

Comment:
```suggestion
                mc.getItemRenderer().getModel(itemStack, null, mc.player));
```
out of changed lines but just saw that